addons/mail/static/src/discuss/gif_picker/gif_picker.xml

```diff
@@ -0,0 +1,79 @@
+                    <span class="fs-5 text-muted d-block">Failed to load gifs...</span>
+                </div>
+                <t t-if="state.showCategories">
+                    <div t-if="!state.loadingError" class="o-discuss-gif-categories row row-cols-2 gy-2 gx-2">
+                        <div t-if="!store.guest" class="col">
+                            <div class="o-discuss-gif-category position-relative ratio ratio-16x9" t-on-click="onClickFavoriteCategory">
+                                <img
+                                    t-if="state.favorites.length > 0"
+                                    class="o-discuss-gif-category"
+                                    t-att-src="state.favorites[0].media_formats.tinygif.url"
+                                    loading="lazy"
+                                />
+                                <div class="position-absolute w-100 h-100 top-0 start-0 text-center text-white align-middle fs-2 bg-dark bg-opacity-50">
+                                    Favorites
+                                </div>
+                            </div>
+                        </div>
+                        <t t-foreach="state.categories" t-as="category" t-key="category.name">
+                            <div class="col">
+                                <div class="o-discuss-gif-category position-relative ratio ratio-16x9" t-on-click="() => this.onClickCategory(category)">
+                                    <img class="o-discuss-gif-category-img img-fluid" t-att-src="category.image" loading="lazy"/>
+                                    <div class="position-absolute w-100 h-100 top-0 start-0 text-center text-white align-middle fs-2 bg-dark bg-opacity-50">
+                                        <t t-out="category.name"/>
+                                    </div>
+                                </div>
+                            </div>
+                        </t>
+                    </div>
+                </t>
+                <t t-if="state.gifs and !state.showCategories">
+                    <div class="d-flex gap-1">
+                        <div>
+                            <t t-foreach="evenGif" t-as="gif" t-key="gif_index">
+                                <t t-call="discuss.Gif"/>
+                            </t>
+                        </div>
+                        <div>
+                            <t t-foreach="oddGif" t-as="gif" t-key="gif_index">
+                                <t t-call="discuss.Gif"/>
+                            </t>
+                        </div>
+                    </div>
+                    <div t-if="state.loadingGif" class="text-center fs-6">
+                        <i class="fa fa-spin fa-circle-o-notch"/>
+                    </div>
+                </t>
+            </div>
+        </div>
+    </t>
+
+    <t t-name="discuss.Gif" owl="1">
+        <div class="o-discuss-gif position-relative mt-1 fs-5">
```

Comment:
I don't mind that it's a `t-name` rather than a component. However define classname as-if it was a component.
So `o-discuss-Gif` (PascalCase for name).